docs/content/developers/design-keuzes.md

```diff
@@ -233,9 +233,10 @@ Een voorbeeld response is dan:
 {
     "type": "URI: https://ref.tst.vng.cloud/ref/fouten/ValidationError/",
     "title": "Ongeldige gegevens",
-    "status": 400,
+
+"status": 400,
     "code": "invalid",
-    "detail": "Er deden zich validatiefouten voor",
+e    "detail": "Er deden zich validatiefouten voor",
```

Comment:
waar komt deze `e` vandaan?

Reply:
Heb ik dat gedaan? Zo ja, dan is dat per ongeluk gebeurt door onhandigheid met MarkDown. Mijn intentie was om alleen de sectie "Vertaling van relaties" toe te voegen en de rest van deze notitie intact te laten. Hoe dan ook deze verschillen worden van zelf opgelost als we deze branch gaan mergen met die van jou.

Reply:
Ik ben ervan overtuigd dat het per ongeluk gebeurd is, en we gaan het inderdaad oplossen, maar dan heb ik het nu via de review in ieder geval op mijn radar staan.